homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
 SERVICE_SYNC = 'sync'
 
+DEFAULT_NUM_REPEATS = '1'
```

Comment:
You are correct, but letting this be specified as a number allows for much easier integration. If I'm trying to trigger something through IFTTT, I can't generate a list of commands on the fly. I can, however, take the number given to me and pass that on to home assistant.